projects/plugins/wpcomsh/.eslintrc.js

```diff
@@ -3,6 +3,7 @@ const loadIgnorePatterns = require( 'jetpack-js-tools/load-eslint-ignore.js' );
 module.exports = {
 	root: true,
```

Comment:
We should drop the `root: true`, and then anything else that's reproducing the default. This should be all that's needed:
```js
module.exports = {
	rules: {
		// Enforce use of the correct textdomain.
		'@wordpress/i18n-text-domain': [
			'error',
			{
				allowedTextDomain: 'wpcomsh',
			},
		],

		'jest/no-disabled-tests': 'warn',
	},
};
```